js/hotgraphicPopupView.js

```diff
@@ -0,0 +1,188 @@
+define([
+    'core/js/adapt'
+], function(Adapt) {
+    'use strict';
+
+    var HotgraphicPopupView = Backbone.View.extend({
+
+        className: 'hotgraphic-popup',
+
+        attributes: function() {
+            return {
+                'role': 'dialog',
+                'aria-label': Adapt.course.get('_globals')._components._hotgraphic.ariaPopupLabel
+            };
+        },
```

Comment:
can you remove this accessibility stuff?
You get two popup opened messages if this is here. one from the notify and one from the hotgrahic,